### ISSUE TITLE
Issue with Partials and a11y, voiceover

### ISSUE THREAD

**Issue by bluSCALE4** [wycats/handlebars.js#1265]
Working on a task using a customized version of Handlebars 1.3.0. Noticed that when using a partial with input[type=checkbox], voiceover fails to pick up the label. Using aria-labelledby doesn't help. The only solution, I've found, is to place the inputs in the main hbs that's passed into compile. 

Has this been remedied down the road?
